common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -45,7 +45,8 @@
             renderCategoryMap: function(map) {
                 var categoryTemplate = edx.HtmlUtils.template($('#new-post-menu-category-template').html()),
                     entryTemplate = edx.HtmlUtils.template($('#new-post-menu-entry-template').html()),
-                    mappedCategorySnippets = _.map(map.children, function(name) {
+                    mappedCategorySnippets = _.map(map.children, function(nameType) {
```

Comment:
@bjacobel:  Thanks.  This would result in a lot of changes to test data (see actual review).  Could you state whether this should be a requirement for thumbs, or if we should leave this as-is, and simply let it be a lesson learned for the future?